frontend/src/assets/js/main.js

```diff
@@ -12,11 +12,19 @@ function generateRoomCode(length = 4) {
 let collaborators = [];
 let code = "";
 let connected = false;
-let websocket;
```

Comment:
Might still need this as the `window.handleContentChange` requires the websocket.